Should create output directory before saving result

You should create the output directory inside go before saving result since results folder does not exist when we pull from git.

```
2023/06/12 12:06:24 Encoding image 1
panic: open results/500/1.png: no such file or directory

goroutine 1 [running]:
```

--- a/main.go
+++ b/main.go
@@ -45,7 +45,7 @@ func parseImageConfigArgs() {
 
 func generateIagesFromLocations(locs LocationsFile) {
 	if _, err := os.Stat("results/" + strconv.Itoa(imgConf.MaxIter)); os.IsNotExist(err) {
-		os.Mkdir("results/"+strconv.Itoa(imgConf.MaxIter), 0755)
+		os.MkdirAll("results/"+strconv.Itoa(imgConf.MaxIter), 0755)
 	}
 
 	for index, loc := range locs.Locations {

